packages/transport/test/modules/messages/MessageController.test.ts

```diff
@@ -226,8 +226,8 @@ describe("MessageController", function () {
             await TestUtil.terminateRelationship(sender, recipient);
         });
 
-        test("should not send a message on a terminated relationship", async function () {
-            await expect(TestUtil.sendMessage(sender, recipient)).rejects.toThrow("error.transport.messages.missingOrInactiveRelationship");
+        test("should be able to send a message on a terminated relationship", async function () {
+            await expect(TestUtil.sendMessage(sender, recipient)).resolves.not.toThrow();
         });
```

Comment:
this is not enough testing. You should also check that the message is received when revived

Reply:
also I'd like to see a test about identityDeletion in transport